app/src/userpages/components/StreamPage/Show/InfoView/index.jsx

```diff
@@ -24,6 +23,7 @@ type Props = {
 export const InfoView = ({ disabled }: Props) => {
     const stream = useSelector(selectEditedStream)
     const dispatch = useDispatch()
+    /* eslint-disable-next-line no-unused-vars */
```

Comment:
Or just leave out `isCopied`? -> `const { copy } = useCopy()`

Reply:
Oh, I thought it was returning an array 🤦‍♂